edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -75,18 +75,21 @@ public class Hosting
                 }
             };
 
-            int port = configuration.GetValue("httpSettings:port", 443);
+            int mainPort = configuration.GetValue("httpSettings:port", 443);
+            int metricsPort = configuration.GetValue("httpSettings:metrics_port", 9600);
             IWebHostBuilder webHostBuilder = new WebHostBuilder()
                 .UseKestrel(
                     options =>
                     {
                         options.Listen(
                             !Socket.OSSupportsIPv6 ? IPAddress.Any : IPAddress.IPv6Any,
-                            port,
+                            mainPort,
                             listenOptions =>
                             {
                                 listenOptions.UseHttps(connectionAdapterOptions);
                             });
+
+                        options.Listen(!Socket.OSSupportsIPv6 ? IPAddress.Any : IPAddress.IPv6Any, metricsPort);
```

Comment:
How was it listening on the metrics port before?

Reply:
It started its own server

Reply:
From the way UseHttps() is used above, I would think that https isn't configured for the metrics port. From other conversations though, you think it is? Didn't you say that https worked for direct methods on 9600 /twin?